### ISSUE TITLE
Unable to find draggable with id: item1, please help me

### ISSUE THREAD

**Issue by sairede** [atlassian/react-beautiful-dnd#2393]
`<Row gutter={[24, 24]}>
        <DragDropContext onDragEnd={onDragEnd}>
          {data?.lists.map((item) => (
            <Col span={6}>
              <DraggableElement
                elements={elements[item]}
                key={item}
                listTitle={item}
              />
            </Col>
          ))}
        </DragDropContext>
      </Row>

<Row className="drop-styles">
      <Col span={24}>{listTitle}</Col>
      <Droppable droppableId={`${listTitle}`}>
        {(provided) => (
          <Col span={24} {...provided.droppableProps} ref={provided.innerRef}>
            {elements.map((item, index) => (
              <ListItem key={item.id} item={item} index={index} />
            ))}
            {provided.placeholder}
          </Col>
        )}
      </Droppable>
      <Text>+Add new card</Text>
    </Row>

<Draggable draggableId={item.id} index={index} key={item.id}>
      {(provided, snapshot) => {
        return (
          <Row
            className="drag-item"
            ref={provided.innerRef}
            snapshot={snapshot}
            {...provided.draggableProps}
            {...provided.dragHandleProps}
          >
            <Col span={24}>Content</Col>
            <Col span={24}>
              <Row>
                <Col span={16}>{item.content}</Col>
                <Col span={8} style={{ textAlign: "right" }}>
                  <Avatar.Group size="small">
                    {item.members.map((person) => (
                      <Tooltip title={person?.name} placement="top">
                        <Avatar src={person.image} />
                      </Tooltip>
                    ))}
                  </Avatar.Group>
                </Col>
              </Row>
            </Col>
          </Row>
        );
      }}
    </Draggable>`

**Comment by rizwanliaqat97** [comment]
@sairede try removing react's strict mode

**Comment by sairede** [comment]
Thanks @rizwanliaqat97 it worked

**Comment by heygema** [comment]
@rizwanliaqat97 it works, thanks

**Comment by chimaOG** [comment]
@rizwanliaqat97 , tried it and it worked for me as well

**Comment by syafilm** [comment]
@rizwanliaqat97  Oh my goodness it works, thank you

**Comment by varnitgoyal2** [comment]
@rizwanliaqat97 omg it works!!!! thankyou

**Comment by Patil-Mayur** [comment]
Can someone check this issue please.
I am not able to understand what is the issue in this example
https://codesandbox.io/s/staging-field-ej6ghv?file=/src/App.js

**Comment by rizwanliaqat97** [comment]
> Can someone check this issue please. I am not able to understand what is the issue in this example https://codesandbox.io/s/staging-field-ej6ghv?file=/src/App.js

@Patil-Mayur in your `index.js` file change this...

```javascript
root.render(
  <StrictMode>
    <App />
  </StrictMode>
);
```

into this...

```javascript
root.render(<App />);
```

like I said above, just remove the react's strict mode

**Comment by Amzat19** [comment]
Thanks @rizwanliaqat97. This saved my day!.

**Comment by CruelFatOtaku** [comment]
@rizwanliaqat97 It worked !!!

**Comment by jessicaribeiroalves** [comment]
I was facing the same issue and when I removed the `StrictMode` as suggested it worked but then I found a repo which was using the `StrictMode` but not using `root.render` and it also worked for me.

So in the `App.js` file, instead of using this:
```
const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
    <React.StrictMode>
        <App />
    </React.StrictMode>
);
```
I replaced with:
```
ReactDOM.render(
    <React.StrictMode>
        <App />
    </React.StrictMode>,
    document.getElementById('root')
);
```

**Comment by imyuanli** [comment]
> @sairede try removing react's strict mode
> 
> Not a fix, but it helped me. Hopefully works for you too.

thanks. It's very helpful to me!!!

**Comment by Audiopolis** [comment]
@jessicaribeiroalves That will disable concurrent features in React 18: Unable to find draggable with id

StrictMode also serves a purpose. It would be nice to see a fix for this that works with StrictMode.

**Comment by jessicaribeiroalves** [comment]
@Audiopolis yep you're right. I was trying to avoid removing the `StrictMode` while there's no fix for this bug. But the solution I suggested also has side effects. Anyways I think we'll have to wait for a fix for it to work properly.

**Comment by romerobrjp** [comment]
We are in 2023... Any fix so far? 
We should not have to remove StrictMode, it sounds like a "gambiarra" (google it)...

**Comment by Audiopolis** [comment]
> We are in 2023... Any fix so far? We should not have to remove StrictMode, it sounds like a "gambiarra" (google it)...

The maintenance notice in the README seems to indicate that this will never be fixed.

**Comment by suissa** [comment]
> We are in 2023... Any fix so far? We should not have to remove StrictMode, it sounds like a "gambiarra" (google it)...

I put a key in Droppable (like the name of the column)

ps: brasila aqui tb

**Comment by Affaq-Ahmed** [comment]
Instead of Removing the Strict Mode, you can use this approach:

```
import { useEffect, useState } from "react";
import { Droppable } from "react-beautiful-dnd";

export const StrictModeDroppable = ({ children, ...props }) => {
  const [enabled, setEnabled] = useState(false);

  useEffect(() => {
    const animation = requestAnimationFrame(() => setEnabled(true));

    return () => {
      cancelAnimationFrame(animation);
      setEnabled(false);
    };
  }, []);

  if (!enabled) {
    return null;
  }

  return <Droppable {...props}>{children}</Droppable>;
};
```
